src/containers/LoginForm.jsx

```diff
@@ -0,0 +1,166 @@
+  };
+
+  handleSubmit = (event) => {
+    const { email, password } = this.state;
+    event.preventDefault();
+    const formErrors = validateLoginForm(email, password);
+    if (formErrors.length === 0) {
+      this.setState({
+        formErrors: [],
+      });
+      const data = { email, password };
+      store.dispatch({ type: IS_LOADING, payload: true });
+      this.props.loginAction(data);
+    } else {
+      this.setState({
+        formErrors,
+        touched: false,
+      });
+    }
+  };
+
+  swapModal = () => {
+    store.dispatch({ type: REGISTER });
+  };
+
+  render() {
+    const { loginUser } = this.props;
+    const { errors, loading } = loginUser;
+    const { formErrors, touched, show } = this.state;
+    return (
+      <div>
+        <Form className="main-container" onSubmit={this.handleSubmit}>
+          <Form.Group name="email">
+            {errors && (
+              <Alert
+                className="close-login-alert"
+                dismissible
+                variant="danger"
+                show={show}
+                onClick={this.onAlertClose}
+              >
+                <p>
+                  <small>{errors}</small>
+                </p>
+              </Alert>
+            )}
+            <Form.Label htmlFor="email">Email address</Form.Label>
+            <div className="text-danger email-warning">
+              {formErrors[0] === 'email' && !touched && formErrors[1]}
+            </div>
+            <p className="text-danger email-warning">{errors.email}</p>
+            <Form.Control
+              className={
+                formErrors[0] === 'email' ? 'red-form-border' : 'email-input'
+              }
+              name="email"
+              type="email"
+              placeholder="Enter your email"
+              onChange={this.handleChange}
+            />
+          </Form.Group>
+          <Form.Group name="password">
+            <Form.Label htmlFor="password">Password</Form.Label>
+            <div className="text-danger password-warning">
+              {formErrors[0] === 'password' && !touched && formErrors[1]}
+            </div>
+            <p className="text-danger password-warning">{errors.password}</p>
+            <Form.Control
+              className={
+                formErrors[0] === 'password'
+                  ? 'red-form-border'
+                  : 'password-input'
+              }
+              name="password"
+              type="password"
+              placeholder="Password"
+              onChange={this.handleChange}
+            />
+          </Form.Group>
+          {loading ? (
+            <Button className="btn-disabled btn-one">
+              <Spinner
+                as="span"
+                animation="grow"
+                size="sm"
+                role="status"
+                aria-hidden="true"
+                className="btn-disabled"
+              />
+              Loading...
+            </Button>
```

Comment:
@MandelaK, you can do this in a cleaner way, sync with @ElMonstro to help you out on this.